actionview/lib/action_view/template/handlers/erb.rb

```diff
@@ -73,6 +70,9 @@ def flush_newline_if_pending(src)
         end
       end
 
+      # For backwards compatibility
+      Erubis = Erubi
```

Comment:
Looks like both `add_preamble` and `add_text` were public. But I don't if we need to deprecate those two or other methods `::Erubis::Eruby` might have had. cc @rafaelfranca